plugins/system/debug/debug.php

```diff
@@ -662,8 +664,8 @@ protected function displayProfileInformation()
 			);
 		}
 
-		$avgTime = $totalTime / count($marks);
-		$avgMem  = $totalMem / count($marks);
+		$avgTime = $totalTime / max(array(count($marks), 1));
+		$avgMem  = $totalMem / max(array(count($marks), 1));
```

Comment:
```suggestion
-		$avgTime = $totalTime / max(array(count($marks), 1));
-		$avgMem  = $totalMem / max(array(count($marks), 1));
+		$avgTime = $totalTime / max(count($marks), 1);
+		$avgMem  = $totalMem / max(count($marks), 1);
```
I suggest doing it a little shorter.